eclass/perl-module.eclass

```diff
@@ -296,6 +296,8 @@ perl-module_src_configure() {
 			RANLIB="$(tc-getRANLIB)" \
 			OPTIMIZE="${CFLAGS}" \
 			LDFLAGS="${LDFLAGS}" \
+			CCFLAGS="${CFLAGS}" \
```

Comment:
Are you sure this isn't going to cause problems like the ones mentioned in 69d4cc312072ad28491d2dce5798dd49d63713e5?